ApiBundle/Controller/ContentController.php

```diff
@@ -113,7 +114,7 @@ public function listAction(Request $request)
         );
 
         $repository =  $this->get('open_orchestra_model.repository.content');
-        $contentCollection = $repository->findByContentTypeInLastVersionForPaginateAndSearch($contentType, $columnsNameToEntityAttribute, $columns, $search, $order, $skip, $limit);
+        $contentCollection = $repository->findByContentTypeInLastVersionForPaginateAndSearchAndSiteId($contentType, $columnsNameToEntityAttribute, $columns, $search, $siteId, $order, $skip, $limit);
```

Comment:
I don't make this a 'PR-stopper' as we should have a general brainstorming on the naming of the repo methods.